replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
```

Comment:
is this file your replication and extension? the file name makes me think it's just for the extension, but I couldn't find a file for just the replication.  A clearer filename would help.